File: Hacker Rank/SimpleArraySum.java
import java.util.Scanner;
public class Practice3{
    public static void main(String args[]) {
        
        Scanner sc = new Scanner(System.in);
    
        int n = sc.nextInt();
        
        int [] arr = new int[n];
        
        int sum =0;
       
        for (int i = 0; i < n; i++) {
            arr[i]= sc.nextInt();
            
            sum=sum+arr[i];
        }
        
        System.out.println(sum);
    }
}

File: Hacker Rank/SimpleArraySum.kt
import java.util.*

fun main(args: Array<String>) {

    val sc = Scanner(System.`in`)
    var size = sc.nextInt()


    var num = IntArray(size)

    for(i in 0 .. (size-1))
        num[i] = sc.nextInt()

    var sum =0

    for (a in num)
        sum+=a

    print(sum)
}
